packages/ui/src/primitives/tailwind/Input/index.tsx

```diff
@@ -97,7 +97,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
     }
 
     const twcontainerClassName = twMerge(
-      'flex w-full flex-col items-center ',
+      'flex flex-col items-center overflow-hidden',
```

Comment:
There are some distortions like the focus in the admin panel. Primitive behaviour like full width needs to be changed with every component using it.
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/2b14e310-0174-473a-8a79-1db1efbb2d9a)